docs/pages/classic/building-standalone-apps.md

```diff
@@ -124,65 +145,85 @@ Note: Expo does not keep your Apple ID or your Apple ID password.
   I will provide all the credentials and files needed, Expo does limited validation
 ```
 
-Unless you're very familiar with iOS credentials already, it's best to let Expo handle the creation & management of all your credentials for you. If you'd like to know more about iOS credentials, we've written a guide with everything you need to know [here](/app-signing/app-credentials.md).
+If you are unfamiliar with iOS credentials, it is recommended to let Expo handle the creation & management of credentials. To know more about them, refer to the [App Credentials documentation](/app-signing/app-credentials).
 
-If you plan on providing your own certificates, we recommend creating them in the [Apple Developer Portal](https://developer.apple.com/account/resources/certificates/list).
+If you are familiar with iOS credentials and plan to provide your own, use the [Apple Developer Portal](https://developer.apple.com/account/resources/certificates/list) to create them.
 
 #### Enterprise distribution
 
-The Expo build service supports both normal App Store distribution as well as enterprise distribution. To use the latter, you must be a member of the ["Apple Developer Enterprise Program"](https://developer.apple.com/programs/enterprise/). Only normal Apple Developer accounts can build apps that can be submitted to the Apple App Store, and only enterprise developer accounts can build apps that can be distributed using enterprise distribution methods. When you call `expo build:ios`, you just need to choose the correct team, it will be labeled `(In-House)`.
+To build and distribute your app using Apple's Enterprise distribution, it is required that you are a member of the [Apple Developer Enterprise Program](https://developer.apple.com/programs/enterprise/).
+
+Expo Build Service supports Enterprise distribution. When you call `expo build:ios`, you just need to choose the correct team that is labeled with `(In-House)`.
 
 #### Adhoc distribution
 
-> Since this requires you to provide credential overrides, this feature should only be used if you're comfortable with iOS credentials.
+> This distribution requires you to override and use your own credentials credential overrides. Use this distribution if you know how to generate iOS credentials.
+
+One of the easiest ways to test your standalone app is using a [simulator build](#building-for-ios) and then using TestFlight for your physical device for testing. However, if you want to build and install an IPA directly on a physical device through Xcode, follow the steps below to generate one using Expo CLI:
```

Comment:
```suggestion
One of the easiest ways to test your standalone app is using a [simulator build](#building-for-ios) and then using TestFlight for your physical device for testing. However, if you want to build and install an app directly on a physical device through Xcode, follow the steps below to generate one using Expo CLI:
```
Another styleguide point. We should say "app" when referring to a .ipa .apk or .aab file. We should use ".ipa" when we want to specifically talk about the extension.